uber/utils.py

```diff
@@ -477,6 +477,31 @@ def get_real_badge_type(badge_type):
     return c.ATTENDEE_BADGE if badge_type in [c.PSEUDO_DEALER_BADGE, c.PSEUDO_GROUP_BADGE] else badge_type
 
 
+def add_opt(opts, other):
+    """
+    Add an option to a comma-separated string.
+    """
+    other = [other] if isinstance(other, str) else other
+    list = opts.split(',') if opts else []
+    list.extend(other)
```

Comment:
`extend` is nice because it lets us, if we want to, add multiple ribbons at a time by passing a list into this function. It iterates over the items in the list and appends them one by one. For that reason, we needed to convert `other` to a list, lest we end up with something like `['value1', 'value2', 'v', 'a', 'l', 'u', 'e', '3']`, because strings are iterable.